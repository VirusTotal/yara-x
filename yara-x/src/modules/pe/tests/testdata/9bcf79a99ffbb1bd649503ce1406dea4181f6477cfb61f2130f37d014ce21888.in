:100000004D5A90000300000004000000FFFF0000B4
:10001000B8000000000000004000000000000000E8
:1000200000000000000000000000000000000000D0
:100030000000000000000000000000008000000040
:100040000E1FBA0E00B409CD21B8014CCD21546861
:1000500069732070726F6772616D2063616E6E6F7D
:10006000742062652072756E20696E20444F5320A3
:100070006D6F64652E0D0D0A240000000000000065
:10008000504500004C0103006926BECC0000000072
:1000900000000000E00022200B013000000A0000F8
:1000A0000006000000000000F2290000002000000F
:1000B00000400000000000100020000000020000CE
:1000C0000400000000000000040000000000000028
:1000D00000800000000200000000000003004085D6
:1000E00000001000001000000000100000100000D0
:1000F00000000000100000000000000000000000F0
:100100009F2900004F00000000400000080300008D
:1001100000000000000000000000000000000000DF
:10012000006000000C000000D02800005400000017
:1001300000000000000000000000000000000000BF
:1001400000000000000000000000000000000000AF
:100150000000000000000000002000000800000077
:10016000000000000000000008200000480000001F
:1001700000000000000000002E746578740000008C
:10018000F809000000200000000A00000002000042
:1001900000000000000000000000000020000060DF
:1001A0002E7273726300000008030000004000001C
:1001B00000040000000C000000000000000000002F
:1001C00000000000400000402E72656C6F6300006C
:1001D0000C000000006000000002000000100000A1
:1001E000000000000000000000000000400000428D
:1001F00000000000000000000000000000000000FF
:10020000D3290000000000004800000002000500A3
:10021000E4200000EC0700000100000000000000E6
:1002200000000000000000000000000000000000CE
:1002300000000000000000000000000000000000BE
:1002400000000000000000000000000000000000AE
:100250001A730500000A7A1A730500000A7A1A73E5
:100260000500000A7A1A730500000A7A1A7305005D
:10027000000A7A1A730500000A7A1A730500000A48
:100280007A1A730500000A7A1A730500000A7A1AAE
:10029000730500000A7A1A730500000A7A1A7305BA
:1002A00000000A7A1A730500000A7A1A7305000022
:1002B0000A7A1A730500000A7A1A730500000A7A8E
:1002C0001A730500000A7A1A730500000A7A1A7375
:1002D0000500000A7A1A730500000A7A1E02280631
:1002E00000000A2A42534A420100010000000000B7
:1002F0000C00000076342E302E33303331390000BC
:10030000000005006C000000E8020000237E0000F1
:1003100054030000EC03000023537472696E67738A
:1003200000000000400700000400000023555300B7
:100330004407000010000000234755494400000016
:10034000540700009800000023426C6F6200000018
:1003500000000000020000014717A0010900000092
:1003600000FA013300160000010000000A0000003E
:1003700002000000150000000D0000000200000057
:10038000060000000400000001000000030000005F
:100390000400000001000000030000000000B6019E
:1003A0000100000000000600320183020600520135
:1003B00083020600020170020F00A3020000060083
:1003C0001601830206003703DB010A00DA0076011A
:1003D0000E00EC0076010E009E0176010600F4018D
:1003E000DB0100000000010000000000010001002E
:1003F00000001000ED0276011900010001005020FC
:1004000000000000E6019B011E00010057200000D3
:100410000000E601FE02060002005E20000000006F
:10042000E601590306000200652000000000E60115
:100430005302060002006C2000000000E6018E005E
:1004400010000200732000000000E60112031000FB
:1004500003007A2000000000E601F90024000400F7
:10046000812000000000E601E2012400060088204F
:1004700000000000E6096B032A0008008F2000003E
:100480000000E60923032E0008009620000000006B
:10049000E609BF032E0008009D2000000000E609C9
:1004A000D40332000800A42000000000E60123026B
:1004B00006000900AB2000000000E6013C0210002D
:1004C0000900B22000000000E6017A0310000A00D3
:1004D000B92000000000E601980324000B00C020B2
:1004E00000000000E601450037000D00C7200000B5
:1004F0000000E601CF0206000E00CE200000000042
:10050000E601B20206000E00D52000000000E60160
:100510003E0306000E00DC200000000086186A0280
:1005200006000E00000001008D0100000100A50082
:10053000000001001E02000001001800000002007F
:100540007100000001001800000002000A00000015
:1005500001007001000001001E02000001006700A0
:1005600000000100670000000200B4030000010069
:10057000670002001D000200210009006A0201005C
:1005800011006A02060019006A020A0029006A02C4
:10059000100051006A02060031006A0206002E00B7
:1005A0000B0044002E0013004D002E001B006C00B9
:1005B0002E00230075000200010000006F033C00C4
:1005C0000000270340000000D8034000020009009B
:1005D000030002000A00050002000B0007000100F2
:1005E0000C00070004800000000000000000000074
:1005F00000000000000024000000020000000000D5
:100600000000000000001500820000000000000053
:10061000000000000000000000000000B900000021
:1006200000000000000000000000000000000000CA
:100630000C02000000000000003C4D6F64756C650A
:100640003E007472616E73616374696F6E49440039
:1006500070726F647563744A534F4E00556E6974BF
:1006600079456E67696E652E50757263686173694E
:100670006E672E4170706C655374756200476574C7
:100680005472616E73616374696F6E5265636569FC
:100690007074466F7250726F647563744964007051
:1006A000726F64756374496400646576656C6F701D
:1006B00065725061796C6F6164006E65747374610A
:1006C0006E64617264005365744170706C6963613B
:1006D00074696F6E557365726E616D6500617070DF
:1006E0006C69636174696F6E557365726E616D6577
:1006F00000556E697479456E67696E652E50757226
:1007000063686173696E672E4170706C65436F72C8
:100710006500494E61746976654170706C6553740B
:100720006F726500494E617469766553746F7265C6
:100730000050757263686173650044656275676730
:1007400061626C6541747472696275746500496EAA
:100750007465726E616C7356697369626C65546F0F
:1007600041747472696275746500436F6D70696C71
:100770006174696F6E52656C61786174696F6E73D4
:100780004174747269627574650052756E74696D36
:1007900065436F6D7061746962696C6974794174E5
:1007A000747269627574650076616C756500556E6A
:1007B000697479456E67696E652E507572636861FC
:1007C00073696E67004173796E6343616C6C62613B
:1007D000636B00536574556E697479507572636804
:1007E0006173696E6743616C6C6261636B00556E27
:1007F000697479456E67696E652E507572636861BC
:1008000073696E672E4170706C65537475622E64E7
:100810006C6C0053797374656D0046696E6973681A
:100820005472616E73616374696F6E004E6F7449C8
:100830006D706C656D656E74656445786365707424
:10084000696F6E0050757263686173696E672E43DD
:100850006F6D6D6F6E006A736F6E004665746368CE
:1008600053746F726550726F6D6F74696F6E4F72F3
:100870006465720053657453746F726550726F6D66
:100880006F74696F6E4F7264657200416464547274
:10089000616E73616374696F6E4F627365727665C2
:1008A00072002E63746F720053797374656D2E44F9
:1008B0006961676E6F7374696373005379737465EC
:1008C0006D2E52756E74696D652E436F6D70696C17
:1008D0006572536572766963657300446562756716
:1008E00067696E674D6F64657300436F6E74696E00
:1008F000756550726F6D6F74696F6E616C50757253
:1009000063686173657300496E74657263657074C2
:1009100050726F6D6F74696F6E616C507572636841
:100920006173657300694F5353746F726542696EEA
:1009300064696E677300526573746F726554726197
:100940006E73616374696F6E73005265747269656A
:10095000766550726F6475637473006765745F6366
:10096000616E4D616B655061796D656E7473004F9A
:10097000626A6563740050726573656E74436F6478
:1009800065526564656D7074696F6E5368656574F2
:10099000005265667265736841707052656365697F
:1009A0007074006765745F6170705265636569702B
:1009B0007400466574636853746F726550726F6D2E
:1009C0006F74696F6E5669736962696C69747900D6
:1009D00053657453746F726550726F6D6F74696F85
:1009E0006E5669736962696C6974790076697369B6
:1009F00062696C697479006765745F73696D756CA1
:100A000061746541736B546F427579007365745FEF
:100A100073696D756C61746541736B546F42757960
:100A20000000000000000000ED96E21ECFA8F54790
:100A300088DE880CB902EAFA0004200101080320CC
:100A40000001052001011111042001010E08CC7BD9
:100A500013FFCD2DDD5105200101122505200201D6
:100A60000E0E0320000E03200002042001010204E8
:100A700020010E0E0328000E0328000208010008C2
:100A800000000000001E01000100540216577261B0
:100A9000704E6F6E457863657074696F6E546872DE
:100AA0006F777301080100020000000000220100BE
:100AB0001D556E697479456E67696E652E50757245
:100AC00063686173696E672E53746F72657300009B
:100AD000000000003CE802F700014D500200000059
:100AE0005400000024290000240B00000000000036
:100AF00000000000010000001300000027000000BB
:100B000078290000780B00000000000000000000C1
:100B100000000000100000000000000000000000C5
:100B2000000000005253445313CC2CB8F637DD4676
:100B30008A4EFBE9CF3D258801000000443A5C5510
:100B40006E6974795C4B656E53686170655C546561
:100B50006D705C556E697479456E67696E652E506F
:100B6000757263686173696E672E4170706C65534E
:100B70007475622E7064620053484132353600133A
:100B8000CC2CB8F637DDC64A4EFBE9CF3D25883C74
:100B9000E80277DC290BD9E95E2DF2941C6CBAC708
:100BA0002900000000000000000000E12900000012
:100BB0002000000000000000000000000000000015
:100BC00000000000000000D3290000000000000029
:100BD00000000000005F436F72446C6C4D61696EF1
:100BE000006D73636F7265652E646C6C00000000AD
:100BF0000000FF25002000100000000000000000A1
:100C000000000000000000000000000000000100E3
:100C1000100000001800008000000000000000002C
:100C20000000000000000100010000003000008012
:100C300000000000000000000000000000000100B3
:100C4000000000004800000058400000AC02000016
:100C50000000000000000000AC023400000056005C
:100C600053005F00560045005200530049004F00FA
:100C70004E005F0049004E0046004F00000000009B
:100C8000BD04EFFE000001000000000000000000B5
:100C900000000000000000003F0000000000000015
:100CA000040000000200000000000000000000003E
:100CB00000000000440000000100560061007200C6
:100CC000460069006C00650049006E0066006F0018
:100CD00000000000240004000000540072006100C5
:100CE0006E0073006C006100740069006F006E009C
:100CF000000000000000B0040C02000001005300DE
:100D00007400720069006E006700460069006C00A4
:100D1000650049006E0066006F000000E8010000F9
:100D2000010030003000300030003000340062003C
:100D3000300000002C0002000100460069006C0039
:100D40006500440065007300630072006900700074
:100D5000740069006F006E000000000020000000B9
:100D6000300008000100460069006C006500560074
:100D700065007200730069006F006E0000000000E3
:100D800030002E0030002E0030002E003000000019
:100D90006A002500010049006E00740065007200C1
:100DA0006E0061006C004E0061006D006500000087
:100DB00055006E0069007400790045006E00670000
:100DC00069006E0065002E0050007500720063001F
:100DD00068006100730069006E0067002E0041002A
:100DE000700070006C0065005300740075006200B4
:100DF0002E0064006C006C0000000000280002005F
:100E000001004C006500670061006C0043006F004A
:100E100070007900720069006700680074000000CB
:100E2000200000007200250001004F0072006900E0
:100E3000670069006E0061006C00460069006C008C
:100E400065006E0061006D006500000055006E00D9
:100E500069007400790045006E00670069006E004B
:100E600065002E005000750072006300680061008C
:100E7000730069006E0067002E0041007000700072
:100E80006C00650053007400750062002E00640061
:100E90006C006C00000000003400080001005000ED
:100EA00072006F00640075006300740056006500F6
:100EB0007200730069006F006E00000030002E00A9
:100EC00030002E0030002E003000000038000800F6
:100ED000010041007300730065006D0062006C004A
:100EE00079002000560065007200730069006F00F1
:100EF0006E00000030002E0030002E0030002E006A
:100F000030000000000000000000000000000000B1
:100F100000000000000000000000000000000000D1
:100F200000000000000000000000000000000000C1
:100F300000000000000000000000000000000000B1
:100F400000000000000000000000000000000000A1
:100F50000000000000000000000000000000000091
:100F60000000000000000000000000000000000081
:100F70000000000000000000000000000000000071
:100F80000000000000000000000000000000000061
:100F90000000000000000000000000000000000051
:100FA0000000000000000000000000000000000041
:100FB0000000000000000000000000000000000031
:100FC0000000000000000000000000000000000021
:100FD0000000000000000000000000000000000011
:100FE0000000000000000000000000000000000001
:100FF00000000000000000000000000000000000F1
:10100000002000000C000000F43900000000000087
:1010100000000000000000000000000000000000D0
:1010200000000000000000000000000000000000C0
:1010300000000000000000000000000000000000B0
:1010400000000000000000000000000000000000A0
:101050000000000000000000000000000000000090
:101060000000000000000000000000000000000080
:101070000000000000000000000000000000000070
:101080000000000000000000000000000000000060
:101090000000000000000000000000000000000050
:1010A0000000000000000000000000000000000040
:1010B0000000000000000000000000000000000030
:1010C0000000000000000000000000000000000020
:1010D0000000000000000000000000000000000010
:1010E0000000000000000000000000000000000000
:1010F00000000000000000000000000000000000F0
:1011000000000000000000000000000000000000DF
:1011100000000000000000000000000000000000CF
:1011200000000000000000000000000000000000BF
:1011300000000000000000000000000000000000AF
:10114000000000000000000000000000000000009F
:10115000000000000000000000000000000000008F
:10116000000000000000000000000000000000007F
:10117000000000000000000000000000000000006F
:10118000000000000000000000000000000000005F
:10119000000000000000000000000000000000004F
:1011A000000000000000000000000000000000003F
:1011B000000000000000000000000000000000002F
:1011C000000000000000000000000000000000001F
:1011D000000000000000000000000000000000000F
:1011E00000000000000000000000000000000000FF
:1011F00000000000000000000000000000000000EF
:00000001FF
