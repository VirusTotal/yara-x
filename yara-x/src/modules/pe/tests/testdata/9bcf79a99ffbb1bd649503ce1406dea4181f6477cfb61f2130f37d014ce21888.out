is_pe: true
machine: MACHINE_I386
subsystem: SUBSYSTEM_WINDOWS_CUI
os_version {
  major: 4
  minor: 0
}
subsystem_version {
  major: 4
  minor: 0
}
image_version {
  major: 0
  minor: 0
}
linker_version {
  major: 48
  minor: 0
}
opthdr_magic: IMAGE_NT_OPTIONAL_HDR32_MAGIC
characteristics: 8226
dll_characteristics: 34112
timestamp: 3435013737
image_base: 268435456
checksum: 0
base_of_code: 8192
base_of_data: 16384
entry_point: 3058
entry_point_raw: 10738
section_alignment: 8192
file_alignment: 512
loader_flags: 0
size_of_optional_header: 224
size_of_code: 2560
size_of_initialized_data: 1536
size_of_uninitialized_data: 0
size_of_image: 32768
size_of_headers: 512
size_of_stack_reserve: 1048576
size_of_stack_commit: 4096
size_of_heap_reserve: 1048576
size_of_heap_commit: 4096
pointer_to_symbol_table: 0
win32_version_value: 0
number_of_symbols: 0
number_of_rva_and_sizes: 16
number_of_sections: 3
number_of_imported_functions: 1
number_of_delayed_imported_functions: 0
number_of_resources: 3
number_of_version_infos: 7
number_of_imports: 1
number_of_delayed_imports: 0
number_of_exports: 0
version_info_list {
  key: "FileDescription"
  value: " "
}
version_info_list {
  key: "FileVersion"
  value: "0.0.0.0"
}
version_info_list {
  key: "InternalName"
  value: "UnityEngine.Purchasing.AppleStub.dll"
}
version_info_list {
  key: "LegalCopyright"
  value: " "
}
version_info_list {
  key: "OriginalFilename"
  value: "UnityEngine.Purchasing.AppleStub.dll"
}
version_info_list {
  key: "ProductVersion"
  value: "0.0.0.0"
}
version_info_list {
  key: "Assembly Version"
  value: "0.0.0.0"
}
pdb_path: "D:\\Unity\\KenShape\\Temp\\UnityEngine.Purchasing.AppleStub.pdb"
sections {
  name: ".text"
  full_name: ".text"
  characteristics: 1610612768
  raw_data_size: 2560
  raw_data_offset: 512
  virtual_address: 8192
  virtual_size: 2552
  pointer_to_relocations: 0
  pointer_to_line_numbers: 0
  number_of_relocations: 0
  number_of_line_numbers: 0
}
sections {
  name: ".rsrc"
  full_name: ".rsrc"
  characteristics: 1073741888
  raw_data_size: 1024
  raw_data_offset: 3072
  virtual_address: 16384
  virtual_size: 776
  pointer_to_relocations: 0
  pointer_to_line_numbers: 0
  number_of_relocations: 0
  number_of_line_numbers: 0
}
sections {
  name: ".reloc"
  full_name: ".reloc"
  characteristics: 1107296320
  raw_data_size: 512
  raw_data_offset: 4096
  virtual_address: 24576
  virtual_size: 12
  pointer_to_relocations: 0
  pointer_to_line_numbers: 0
  number_of_relocations: 0
  number_of_line_numbers: 0
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 10655
  size: 79
}
data_directories {
  virtual_address: 16384
  size: 776
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 24576
  size: 12
}
data_directories {
  virtual_address: 10448
  size: 84
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 8192
  size: 8
}
data_directories {
  virtual_address: 0
  size: 0
}
data_directories {
  virtual_address: 8200
  size: 72
}
data_directories {
  virtual_address: 0
  size: 0
}
resources {
  length: 684
  rva: 16472
  offset: 3160
  type: RESOURCE_TYPE_VERSION
  id: 1
  language: 0
}
import_details {
  library_name: "mscoree.dll"
  functions {
    name: "_CorDllMain"
    rva: 8192
  }
}
overlay {
  offset: 0
  size: 0
}
